reactor-core/src/main/java/reactor/core/publisher/FluxConcatMapNoPrefetch.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright (c) 2020-Present Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.core.publisher;
+
+import java.util.Objects;
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicReferenceFieldUpdater;
+import java.util.function.Function;
+
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscription;
+import reactor.core.CoreSubscriber;
+import reactor.core.publisher.FluxConcatMap.ConcatMapInner;
+import reactor.core.publisher.FluxConcatMap.FluxConcatMapSupport;
+import reactor.core.publisher.FluxConcatMap.WeakScalarSubscription;
+import reactor.util.annotation.Nullable;
+import reactor.util.context.Context;
+
+/**
+ * Maps each upstream value into a Publisher and concatenates them into one
+ * sequence of items.
+ *
+ * @param <T> the source value type
+ * @param <R> the output value type
+ *
+ * @see FluxConcatMap
+ */
+final class FluxConcatMapNoPrefetch<T, R> extends InternalFluxOperator<T, R> {
+
+	final Function<? super T, ? extends Publisher<? extends R>> mapper;
+
+	FluxConcatMapNoPrefetch(
+			Flux<? extends T> source,
+			Function<? super T, ? extends Publisher<? extends R>> mapper
+	) {
+		super(source);
+		this.mapper = Objects.requireNonNull(mapper, "mapper");
+	}
+
+	@Override
+	public CoreSubscriber<? super T> subscribeOrReturn(CoreSubscriber<? super R> actual) {
+		if (FluxFlatMap.trySubscribeScalarMap(source, actual, mapper, false, true)) {
```

Comment:
see comment on `Flux` relative to that second boolean (`expectDelayErrors`) being `true`.

Reply:
Seems to be hardcoded `true` here as well:
https://github.com/reactor/reactor-core/blob/cb74c0c377599723f4a51549f042040a555aa9d4/reactor-core/src/main/java/reactor/core/publisher/FluxConcatMap.java#L116

Since the source is Scalar, I guess it doesn't make a difference?

Reply:
good point about the scalar nature 👍